apps/tsilhqotin-language-hub/src/Components/Pages/Apps/AppLinkDisplay.tsx

```diff
@@ -30,7 +29,7 @@ export function AppLinkDisplay({ platform, url }: AppLink) {
     const imageInfo = getImageInfoForPlatform(platform);
     return (
         <div className="appLinkDisplay">
-            <a className="appLink" href={url}>
+            <a className="appLink" href={url} target={'_blank'} rel={'noopener noreferrer'}>
```

Comment:
You should create an `<ExternalLink url={url} />` component so that you have a single source of truth for the target and rel props. If you need to tweak them, you do that in one place.